File: java/com/epicdragonworld/gameserver/network/packets/sendable/DeleteObject.java
package com.epicdragonworld.gameserver.network.packets.sendable;

import com.epicdragonworld.gameserver.model.WorldObject;
import com.epicdragonworld.gameserver.network.SendablePacket;

/**
 * @author Pantelis Andrianakis
 */
public class DeleteObject extends SendablePacket
{
	public DeleteObject(WorldObject object)
	{
		// Send the data.
		writeShort(7); // Packet id.
		writeLong(object.getObjectId()); // ID of object to delete.
	}
}

File: EpicDragonWorldServer/Network/Packets/Sendable/DeleteObject.cs
/**
 * Author: Pantelis Andrianakis
 * Date: November 7th 2018
 */
public class DeleteObject : SendablePacket
{
    public DeleteObject(WorldObject obj)
    {
        // Send the data.
        WriteShort(7); // Packet id.
        WriteLong(obj.GetObjectId()); // ID of object to delete.
    }
}
